xml_converter/test_cases/can_fade_tests/can_fade_is_false.xml

```diff
@@ -0,0 +1,8 @@
+<OverlayData>
+<MarkerCategory  Name="IsFalse">
```

Comment:
Aren't names supposed to be all lower case? I don't actually know if that is a rule or not but I think I see that mostly. idc really.

Reply:
There is no rule they need to be. TacO's documentation states that type and therefore name, "Must not contain any spaces or special characters"

Example code on their page has a name that has capital letters. 

```<OverlayData>

  <MarkerCategory name="Collectible">

    <MarkerCategory name="JournalEntry" iconFile="Data\raid1_achievements.png" behavior="3"/>

  </MarkerCategory>  



</OverlayData>```